addons/plugin_refresher/refresh_button.gd

```diff
@@ -133,7 +132,7 @@ enum MenuAction {
 	SHOW_ENABLE_MENU
 }
 
-func _get_plugin_index_by_id(id: String) -> int:
+func _get_plugin_index_by_id(id: Variant) -> int:
```

Comment:
The argument 1 of the function "_get_plugin_index_by_id()" requires the subtype "String" but the supertype "Variant" was provided.

It seemed better to just accept the metadata Variant value directly here